packages/react/src/components/Copy/Copy.js

```diff
@@ -68,7 +68,6 @@ export default function Copy({
         (!children && (animation ? feedback : other['aria-label'])) || null
       }>
       {children}
-      {animation ? feedback : other['aria-label']}
```

Comment:
I don't think we can remove this line since it announces the feedback to screenreaders IIRC

Reply:
Could we change the label to the button as follows ?
`<button ... aria-label={animation ? feedback : other['aria-label']}> ... </button>`

Reply:
Sorry for the late reply though. I was busy with the classes!

Reply:
I'm not sure if we need to change the component at all since the issue you reported is only found in the website. as I mentioned earlier I believe a font-size style rule is being overwritten in the website stylesheet but I haven't looked into it closely